development_helpers/Dockerfile.dev

```diff
@@ -24,33 +28,26 @@ RUN pip install --no-cache-dir -r requirements.txt \
         fi; \
     done;
 
+# User already exists
+# RUN useradd \
+#     --no-log-init \
+#     --shell /bin/bash \
+#     -u 1001 \
+#     ctfd \
+#     && mkdir -p /var/log/CTFd /var/uploads \
+#     && chown -R 1001:1001 /var/log/CTFd /var/uploads /opt/CTFd \
+#     && chmod +x /opt/CTFd/docker-entrypoint.sh
 
-FROM python:3.9-slim-buster as release
-WORKDIR /opt/CTFd
-
-# hadolint ignore=DL3008
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        libffi6 \
-        libssl1.1 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY --chown=1001:1001 . /opt/CTFd
-
-RUN useradd \
-    --no-log-init \
-    --shell /bin/bash \
-    -u 1001 \
-    ctfd \
-    && mkdir -p /var/log/CTFd /var/uploads \
-    && chown -R 1001:1001 /var/log/CTFd /var/uploads /opt/CTFd \
-    && chmod +x /opt/CTFd/docker-entrypoint.sh
-
-COPY --chown=1001:1001 --from=build /opt/venv /opt/venv
 ENV PATH="/opt/venv/bin:$PATH"
 
+USER root
+RUN chown -R 1001:1001 /var/log/CTFd /var/uploads /opt/CTFd 
+RUN chmod +x /opt/CTFd/plugins/llm_verification/docker-entrypoint.dev.sh
+
 USER 1001
 EXPOSE 8000
+
 # LLMV: Modified entrypoint to use development entrypoint.
-ENTRYPOINT ["/opt/CTFd/CTFd/plugins/llm_verification/development_helpers/docker-entrypoint.dev.sh"]
+# ENTRYPOINT ["/opt/CTFd/plugins/llm_verification/development_helpers/docker-entrypoint.dev.sh"]
+# ENTRYPOINT ["/opt/CTFd/plugins/llm_verification/docker-entrypoint.dev.sh"]
+# ENTRYPOINT ["ls", "-la", "/opt/CTFd/plugins/llm_verification", "&&", "/opt/CTFd/plugins/llm_verification/docker-entrypoint.dev.sh"]
```

Comment:
Ive verified this copies into the image but for some reason it fails to run during docker up, I have no idea why.
```
docker compose -f development_helpers/docker-compose.dev.yml up
```

```
docker run --entrypoint /bin/bash -it --rm development_helpers_ctfd
```